File: Ywy/JAVA/src/level1/InsufficientAmount.java
package level1;

public class InsufficientAmount {

	public InsufficientAmount() {
	}
	
    public long solution(int price, int money, int count) {
        long answer = 0;
        int cnt = 0;
        long temp = 0;
        do {
            temp += price * (++cnt);
        } while(cnt < count);
        answer = temp - money;
        return java.lang.Math.max(answer, 0);
    }
}

File: Andy/src/level1/javascript/InsufficientAmount.js
// 프로그래머스 Level 1,부족한 금액 계산하기
function solution(price, money, count) {
    var answer = 0;

    for(let i=1;i<=count;i++)
        answer += (price*i);
    answer -= money;
    return answer>0?answer:0;
}